core/block/import/importer.go

```diff
@@ -131,7 +131,7 @@ func (i *Import) importFromBuiltinConverter(ctx *session.Context,
 	req *pb.RpcObjectImportRequest,
 	c converter.Converter,
 	progress process.Progress) error {
```

Comment:
Maybe we should use the same form of function definition with big amount of arguments/return values, as in https://github.com/anyproto/anytype-heart/pull/383/files#diff-39551f40c77377af86817ac0f6e0fb01b38738e020f21182f71921214b126038R127 ?
This way reader can easily identify the border between func definition and the very function body